lib/puppet/parser/scope.rb

```diff
@@ -560,7 +560,7 @@ def variable_not_found(name, reason=nil)
   # @see #[]=
   # @api public
   #
-  def [](varname, options={})
+  def [](varname, options = EMPTY_HASH)
```

Comment:
This change can be done to the `#initialize` method too, now that `:namespace` out of the way.